egui/src/response.rs

```diff
@@ -277,6 +277,24 @@ impl Response {
         self.drag_released
     }
 
+    /// Mouse dragged, nevertheless it's stuck on screen's edge or not.
+    pub fn motion_delta(&self) -> Vec2 {
```

Comment:
This is very confusing. What does it return on touch screens? Does it matter if `self.dragged` is true or false? Why does it act differently when compiled for wasm?

NOTE: wasm != web. There are those who run egui inside of wasmtime on desktop.

Reply:
I've changed the docs